### PR TITLE
API - status filter description on listing runs

### PR DESCRIPTION
**Pull request by mhamas**
Add status filter description to endpoint for listing actor and actor…task runs.

@m-murasovs please check carefully as I haven't tested how this looks (didn't find way to preview in VSCode and I don't have edit access to apiary.com). Any tips how to visualize this to sanity check it renders correctly welcome. Thanks!

### CODE REVIEW

File: docs/api_v2/api_v2_reference.apib [hunk 1]
```diff
@@ -790,14 +790,15 @@ array elements.
 By default, the records are sorted by the `startedAt` field in ascending order,
 therefore you can use pagination to incrementally fetch all records while new
 ones are still being created. To sort the records in descending order, use `desc=1`
-parameter.
+parameter. You can also filter runs on status ([available statuses](https://docs.apify.com/actors/running#lifecycle)).
```
Comment (m-murasovs) [inline docs/api_v2/api_v2_reference.apib:793]: here, do you mean: "filter runs **by** status"?

File: docs/api_v2/api_v2_reference.apib [hunk 2]
```diff
@@ -1339,7 +1340,7 @@ array elements.
 By default, the records are sorted by the `startedAt` field in ascending order;
 therefore you can use pagination to incrementally fetch all records while new
 ones are still being created. To sort the records in descending order, use the `desc=1`
-parameter.
+parameter. You can also filter runs on status ([available statuses](https://docs.apify.com/actors/running#lifecycle)).
```
Comment (m-murasovs) [inline docs/api_v2/api_v2_reference.apib:1343]: If you meant "filter runs **by** status" above, please change here, too.

**Comment by m-murasovs** [comment]
I can't add comments there because the text wasn't edited but please add 'status' to the list of params (between 'desc' and 'waitForFinish') on lines 1326 and 776.

Also,
@mtrunkat could we get Matej edit access to Apiary for future needs?

**Review by m-murasovs — APPROVED** [review]

**Comment by m-murasovs** [comment]
Tested on Apiary and it's working good!

**Comment by mhamas** [comment]
@m-murasovs I addressed comments. Please, do not merge until https://bitbucket.org/apifyteam/apify-system/pull-requests/2416 is merged and deployed.

**Comment by m-murasovs** [comment]
@mhamas sure! Let me know when it's good to go, or feel free to merge it yourself, I trust you :)

**Comment by mhamas** [comment]
The api with the changes has been deployed to production by @mtrunkat, hence merging this.

**Pull request merged**